nixos/modules/services/misc/jackett.nix

```
@@ -36,6 +37,65 @@ in
       };
 
       package = mkPackageOption pkgs "jackett" { };
+
+      settings = mkOption {
+        description = "Settings for Jackett";
+        default = {};
+        type = types.submodule {
+          freeformType = settingsFormat.type;
+          options = {
+            APIKey = mkOption {
+              type = types.outOfBand;
+              description = "Path to api key secret file.";
+            };
```

The "file" part of the option should be taken care of by `lib.outOfBand`?

```suggestion
            APIKey = mkOption {
              type = types.outOfBand;
              description = "Secret API Key.";
            };
```

---

Ditto for similar options.